demux/demux_lavf.c

```diff
@@ -767,7 +767,6 @@ static void handle_new_stream(demuxer_t *demuxer, int i)
             const AVDOVIDecoderConfigurationRecord *cfg = (void *) sd;
             MP_VERBOSE(demuxer, "Found Dolby Vision config record: profile "
                        "%d level %d\n", cfg->dv_profile, cfg->dv_level);
-            av_format_inject_global_side_data(avfc);
```

Comment:
nitpick to commit message. Generally we use `demux_lavf`, another demux is redundant. It is only important when file doesn't already have prefix.